quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -99,13 +100,14 @@ public final class RawKeycloakDistribution implements KeycloakDistribution {
     private final OutputConsumer outputConsumer;
 
     public RawKeycloakDistribution(boolean debug, boolean manualStop, boolean enableTls, boolean reCreate, boolean removeBuildOptionsAfterBuild, int requestPort) {
-        this(debug, manualStop, enableTls, reCreate, removeBuildOptionsAfterBuild, requestPort, new DefaultOutputConsumer());
+        this(debug, manualStop, enableTls, null, reCreate, removeBuildOptionsAfterBuild, requestPort, new DefaultOutputConsumer());
     }
 
-    public RawKeycloakDistribution(boolean debug, boolean manualStop, boolean enableTls, boolean reCreate, boolean removeBuildOptionsAfterBuild, int requestPort, OutputConsumer outputConsumer) {
+    public RawKeycloakDistribution(boolean debug, boolean manualStop, boolean enableTls, Path serverKeyStorePath, boolean reCreate, boolean removeBuildOptionsAfterBuild, int requestPort, OutputConsumer outputConsumer) {
```

Comment:
Instead of copying the truststore, wouldn't it be simpler to just set `https-key-store-file` config option?